pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java

```diff
@@ -95,10 +95,17 @@ private static void generateSystemMetrics(SimpleTextOutputStream stream, String
                 stream.write(sample.name);
                 stream.write("{cluster=\"").write(cluster).write('"');
                 for (int j = 0; j < sample.labelNames.size(); j++) {
+                    String labelValue = sample.labelValues.get(j);
+                    if(labelValue != null &&
+                            (labelValue.startsWith("\"") ||
```

Comment:
I don't think we need to check `startsWith` and `endsWith`.

We can just simplify the logic as follows:

```
if (labelValue != null) {
    labelValue = labelValue.replace("\"", "\\\"");
}
```

Reply:
ok, that's more simple! I improve the code.